src/Riot/Collection/LeagueEntryDTOCollection.php

```diff
@@ -26,6 +26,11 @@ public static function createFromArray(array $data): self
     {
         $collection = new self();
         foreach ($data as $item) {
+
+            if (!isset($item['leagueId'])) {
+                continue;
```

Comment:
Maybe instead of skipping this league entry, it would be better to make `leagueId` not required in the DTO? Maybe someone would be interested in the number of wins/losses of the Player in some of the leagues you mentioned.

Reply:
Done